### DISCUSSION TITLE
Apollo on ps5

### DISCUSSION THREAD

**Discussion by reboot89** [bucanero/apollo-ps4#143] [Ideas]
Hi, this is a request for the Apollo save tool on ps5.

Many can't use things like remote play due to staying on older firmware, such as myself being on firmware 4.0.3 on the ps5.

So the offline activation may come handy.
Please if its possible, grant our wish. Thanks in advance.

**Comment by xlj44400** [comment]
it is on ps5debug

**Comment by reboot89** [comment]
@xlj44400 sorry but can you explain? Is it possible to offline activate the ps5 now and enable remote play?
